outgoing_routing/README.rst

```diff
@@ -9,5 +9,8 @@ Picking and Reservation Strategy
 Changelog
 =========
 
+* 1.0.3 (2022-03-31)
+    - [FIX] Fixed reservation and removal strategy on ventor base module ignore force removal strategy on Product Category in Odoo 
```

Comment:
Fixed reservation and removal strategy on ventor base module ignore force removal strategy on Product Category or/and Location

Reply:
Done